sematic/plugins/building/docker_builder.py

```diff
@@ -599,22 +640,38 @@ def _build_image_from_base(
             **optional_kwargs,
         )
 
-    if status_updates is None:
-        logger.warning("Built image '%s' without any response", built_image_name)
-        image = docker_client.images.get(str(effective_base_uri))
-        return image, ImageURI.from_image(image)
+    spinner_process.terminate()
```

Comment:
should probably nest the above in a `try` and put this process cleanup in a `finally` so if there is an exception in the above you don't leave the main process hung because it didn't wait on/clean up its child process.

Reply:
Done.